speech-to-text/src/main/java/com/ibm/watson/developer_cloud/speech_to_text/v1/SpeechToText.java

```diff
@@ -232,6 +233,10 @@ private void buildRecognizeRequest(RequestBuilder requestBuilder, RecognizeOptio
     if (options.customizationId() != null) {
       requestBuilder.query(CUSTOMIZATION_ID, options.customizationId());
     }
+
+    if (options.customizationWeight() != null) {
+      requestBuilder.query(CUSTOMIZATION_WEIGHT, options.customizationWeight().toString());
```

Comment:
We're not using `toString()` on the other `Double` parameters, so maybe we should remove that to stay consistent.